src/transforms/ember-getWithDefault-to-lodash-get/tests/fixtures/lodash.import.output.js

```diff
@@ -0,0 +1,5 @@
+import _, {get as getWithDefault} from 'lodash';
```

Comment:
I think it might be preferred to use `lodash/get` in this case instead of adding onto the `lodash` import
```suggestion
import _ from 'lodash'
import getWithDefault from 'lodash/get';
```

Reply:
Done. If lodash get is not already getting used, then following import statement will be used:

import getWithDefault from 'lodash/get';